cmd/gapit/flags.go

```diff
@@ -322,7 +322,7 @@ type (
 		Capture struct {
 			Frames int `help:"only capture the given number of frames. 0 for all"`
 		}
-		IgnoreAndroidFrameBoundary bool `help:"If true: ignore the ANDROID_frame_boundary extension. If false: respect it."`
+		UseAndroidFrameBoundary bool `help:"If false: ignore the ANDROID_frame_boundary extension. If true: respect it."`
```

Comment:
can you change the help text here to something like 'When enabled, AGI uses ANDROID_frame_boundary extension as frame delimiters. Otherwise, present calls are used. Default is false.' before submitting?